LoopKitUI/Views/ScheduleEditor.swift

```diff
@@ -143,16 +149,31 @@ struct ScheduleEditor<Value: Equatable, ValueContent: View, ValuePicker: View, A
         case .legacySettings:
             return AnyView(wrappedPage)
         case .acceptanceFlow, .settings:
-            return AnyView(configurationPage)
+            return AnyView(contentWithCancel)
```

Comment:
I don't think we want the cancel button to be in the acceptance flow - users should be able to go back and forth between screens (I discussed the "close" button issue with Melissa and Brooke, and we all agreed that the editors are too crowded to have one)

Reply:
Good catch.  I'll fix it! 👍